templates/adp-backend-template-node/skeleton/docker-compose.migrate.yaml

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+x-common-migration: &common-migration
+  POSTGRES_HOST: ${POSTGRES_HOST:-${{ values.project_name }}-postgres}
+  SCHEMA_ROLE: ${POSTGRES_SCHEMA_ROLE:-postgres}
+  SCHEMA_USERNAME: ${POSTGRES_SCHEMA_USERNAME:-postgres}
+  SCHEMA_PASSWORD: ${POSTGRES_SCHEMA_PASSWORD:-ppp}
+  SCHEMA_NAME: ${POSTGRES_SCHEMA_NAME:-public}
+
+x-common-postgres: &common-postgres
+  POSTGRES_PORT: 5432
+  POSTGRES_DB: ${POSTGRES_DB:-${{ values.project_name }}}
+  POSTGRES_PASSWORD: ${POSTGRES_ADMIN_PASSWORD:-ppp}
+  POSTGRES_USERNAME: ${POSTGRES_ADMIN_USERNAME:-postgres}
+
+services:
+  database-up:
+    image: liquibase/liquibase:3.10.x
+    environment:
+      << : [*common-postgres, *common-migration]
+    entrypoint: >
+      sh -c "/scripts/migration/database-up $${SCHEMA_PASSWORD}"
+    depends_on:
+      - ${{ values.project_name }}-postgres
+    volumes:
+      - ./changelog:/liquibase/changelog
+      - ./scripts:/scripts
+
+  database-down:
+    image: liquibase/liquibase:3.10.x
+    environment:
+      << : [*common-postgres, *common-migration]
+    entrypoint: >
+      sh -c "/scripts/migration/database-down"
+    depends_on:
+      - ${{ values.project_name }}-postgres
+    volumes:
+      - ./changelog:/liquibase/changelog
+      - ./scripts:/scripts
+
+  ${{ values.project_name }}-postgres:
+    image: postgres:11.4-alpine
```

Comment:
shouldn't be 11 ?

Reply:
15 / 16?

Reply:
@DaRoszko It is the same as in our demo apps. Also, checked some of the ffc-pay and ffc-ahwr repos. They are using the same.

Reply:
@DaRoszko  one to discuss with John. Most services are still using 11.